src/allmydata/scripts/startstop_node.py

```diff
@@ -6,27 +6,31 @@
 from allmydata.util.encodingutil import listdir_unicode, quote_output
 
 
-class StartOptions(BasedirOptions):
+class StartStopCommonOptions(BasedirOptions):
+    optParameters = [
+        ["pidfile", None, None, "Tell the node which pid file should use."],
+        ]
```

Comment:
I was following the identation on other parts of the code, but I agree with you it's more clear dedenting it.